plugins/eslint-config/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@blockly/eslint-config",
+  "version": "1.20200426.0",
+  "description": "ESlint configuration used by Blockly plugins.",
+  "files": [
+    "index.js"
+  ],
+  "author": "Blockly Team",
+  "keywords": [
+    "blockly",
+    "eslint",
+    "eslintplugin",
+    "dev",
+    "plugin"
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/master/plugins/eslint-config#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "plugins/eslint-config"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
+    "babel-eslint": "^10.1.0",
+    "eslint-config-google": "^0.14.0",
+    "eslint-plugin-jsdoc": "^22.1.0"
+  },
+  "peerDependencies": {
+    "eslint": "6.x",
+    "typescript": "3.x"
+  },
+  "peerDependenciesMeta": {
+    "typescript": {
+      "optional": true
+    }
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://wombat-dressing-room.appspot.com"
+  },
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
Can this be removed?

Reply:
Possibly, do you mean specifically for this package or for all packages?
I hadn't done much digging into whether this is necessary, I just assumed so as it was in the original workspace-search package.json

Reply:
I meant specifically for this package, but thinking about it more it might be good to revisit whether it should be in the template. We had this in workspace search since we planned on supporting node for that package and it was recommended to have a baseline version.

Reply:
Yeah, there's a chance this isn't relevant, but I want to keep our package.jsons consistent (incase someone copies them as an example), so if the template has it (which it currently does), then lets keep it there.